gnovm/pkg/gnomod/pkg.go

```diff
@@ -5,40 +5,32 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
 type Pkg struct {
-	name     string
-	path     string
-	draft    bool
-	requires []string
+	Dir      string   // absolute path to package dir
+	Name     string   // package name
+	Requires []string // dependencies
+	Draft    bool     // whether the package is a draft
+}
+
+type SubPkg struct {
```

Comment:
We shouldn't have anything called Sub*.
From the perspective of Go/Gno, "sub" packages are just independent packages.
We can have maybe one utility function that returns all "sub" Modules inside a dir, but that should be about it. (And btw the filetests folder once we support that, is not itself a module. For safety we should also disallow a module in a folder with that name too).